components/operations-controller/api/v1alpha1/operation_types.go

```diff
@@ -31,13 +31,14 @@ func init() {
 	SchemeBuilder.Register(&Operation{}, &OperationList{})
 }
 
-// +kubebuilder:validation:Enum=Create;Update;Delete
+// +kubebuilder:validation:Enum=Create;Update;Delete;Unpair
 type OperationType string
 
 const (
 	OperationTypeCreate OperationType = "Create"
 	OperationTypeUpdate OperationType = "Update"
 	OperationTypeDelete OperationType = "Delete"
+	OperationTypeUnpair OperationType = "Unpair"
```

Comment:
do we need a separate operation type called unpair or can we treat it as a operation of type update - i noticed that it most switch cases we are anyway using the "update" case ?